# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../esnacc/no-examples.patch.cross
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- SNACC/makefile.vanilla	2008-08-25 07:15:18.000000000 +0200
+++ SNACC/makefile	2008-08-25 07:15:23.000000000 +0200
@@ -3,7 +3,7 @@
 include ./makehead
 
 #-------------------------------------------------------------------------------
-SUBDIRS = c-lib c++-lib compiler c-examples c++-examples
+SUBDIRS = c-lib c++-lib compiler
 
 all: all-redirect 
 	@for i in $(SUBDIRS) ; do \
